README.md

```diff
@@ -37,7 +51,7 @@ az deployment group create -g  myresource-group --template-file .\main.json -p p
 
 Alternatively you can deploy through the Azure Portal directly:
 
-[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fsebader%2Fteams-distributor%2Fmain%2Fmain.json)
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fsebader%2Fteams-distributor%2Fmain%2Fdeployment%2Fmain.json)
```

Comment:
why this change? did you change the file structure? If so, its missing from the PR